extensions/extensions-data-lineage/extensions-data-lineage-http-consumer-service/pom.xml

```diff
@@ -31,7 +31,7 @@
 
   <properties>
 <!--    Overriding cucumber version to align with quarkus-cucumber -->
-    <version.cucumber>7.4.1</version.cucumber>
+    <version.cucumber>${version.quarkus.cucumber.java}</version.cucumber>
```

Comment:
A: Can't we just remove this? We can set the version in dependency management of build-parent, or just inline the reference to `version.quarkus.cucumber.java`.

Reply:
this is no inline reference in this module. This is an overwrite of the parent property definition.